internal/service/anycast/model_proto_onprem_host.go

```diff
@@ -110,6 +120,7 @@ func (m *ProtoOnpremHostModel) Expand(ctx context.Context, diags *diag.Diagnosti
 		Ipv6Address:       flex.ExpandStringPointer(m.Ipv6Address),
```

Comment:
IpAddress, Ipv6Address and other computed fields should not be Expanded

Reply:
The test fails if the name and a few fields are removed.  as discussed and suggested, keeping it in expand